packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -148,7 +150,7 @@ class ResizeHandle extends (Widget as any)<ResizeHandleOptions> {
     this.$element().addClass(RESIZE_HANDLE_CLASS);
     this.$element().toggleClass(RESIZE_HANDLE_RESIZABLE_CLASS, resizable);
     this._toggleDirectionClass();
-    this._setResizeHandleSize();
+    this._updateDimensions();
```

Comment:
Please, to be sure that _updateDimensions doesn't call upon window resizing

Reply:
It is not called on resizing the window - if thats what you mean